js/themeView.js

```diff
@@ -57,70 +55,30 @@ export default class ThemeView extends Backbone.View {
   setBackgroundImage() {
     const backgroundImages = this.model.get('_backgroundImage');
     if (!backgroundImages) return;
-
-    let backgroundImage;
-
-    switch (device.screenSize) {
-      case 'large':
-        backgroundImage = backgroundImages._large;
-        break;
-      case 'medium':
-        backgroundImage = backgroundImages._medium;
-        break;
-      default:
-        backgroundImage = backgroundImages._small;
-    }
-
-    if (backgroundImage) {
-      this.$el.addClass('has-bg-image');
-      this.$background
-        .css('background-image', 'url(' + backgroundImage + ')');
-      return;
-    }
-
-    this.$el.removeClass('has-bg-image');
-    this.$background.css('background-image', '');
+    const backgroundImage = backgroundImages[`_${Adapt.device.screenSize}`] ?? backgroundImages._small;
```

Comment:
Why have we moved away from the device singleton?

Reply:
@cahirodoherty-learningpool 

We've already done this in boxmenu

https://github.com/adaptlearning/adapt-contrib-boxMenu/blob/040166743e2a765a7d08bd5b4e6b08a7705ecf64/js/BoxMenuView.js#L73

Not sure, bit less hard-codey / neater..?

Reply:
Adapt.device > device.